src/index.ts

```diff
@@ -9,3 +9,6 @@ export * from './CohortRepresentations';
 export * from './cohortview';
 export * from './LoginDialog';
 export * from './rest';
+
+export * from './util';
```

Comment:
Maybe you wish to move this files somewhere else so you don't have to export the util folder/ file